bucket/fpcalc.json

```diff
@@ -0,0 +1,25 @@
+{
+    "version": "1.5.1",
+    "description": "fpcalc binary from Chromaprint for generating audio fingerprints used by AcoustID",
+    "homepage": "https://acoustid.org/chromaprint",
+    "license": "MIT|LGPL-2.1-or-later",
```

Comment:
LGPL programs allows for it to be embedded in a larger work and still allows the larger work to be licensed using a different one (even proprietary).

So technically it will be MIT.

But the final verdict is from the authors. Ask them in their Github account if they approve of Chromaprint being included in Scoop as "MIT or LGPL-2.1-or-later".

Reply:
@lalinsky Thoughts on this?